lib/generators/scenic/view/templates/db/migrate/create_view.erb

```diff
@@ -1,5 +1,5 @@
 class <%= migration_class_name %> < ActiveRecord::Migration
   def change
-    create_view :<%= plural_name %><%= ", materialized: true" if materialized? %>
+    create_view "<%= plural_name %>"<%= ", materialized: true" if materialized? %>
```

Comment:
We could do symbols with the quotes, like `:"non_public.searches"`, but without the quotes it ends up as `:non_public.searches`, which is a big no-no. I personally just don't like seeing symbols with quotes, but that's not for any good reason, so I'd be happy to go that route if you want.